### PR TITLE
React 15/16 prep

### PR DESCRIPTION
**Pull request by laurence-hudson-tessella**
As per #1

### CODE REVIEW

**Comment by pke** [comment]
I think the generated code should not be checked in but rather built as pre-publish step?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-delay",
-  "version": "0.0.3",
+  "version": "0.0.4",
```
Comment (pke) [inline package.json:3]: Why not make it a 1.0 right now? What else does this need to be 1.0 aka ready?

**Comment by laurence-hudson-tessella** [comment]
Happy to remove them from the PR, if @chrisshiplet prefers, but the built files are checked in on the base branch, so I updated them for consistency. Also it means I can npm depend on my fork elsewhere to test, without needing to push the fork to the npm repos. I'd also assume this would be ready for a release if chris is happy to merge.

**Comment by laurence-hudson-tessella** [comment]
@chrisshiplet Any thoughts on getting this merged?

**Comment by chrisshiplet** [comment]
@laurence-hudson-tessella @pke Apologies for the super late response, I managed to lose this in a sea of GitHub notifications... Thanks for the contribution! I'll check this branch out and do a little research on best practices for publishing the generated code in the next couple days. I'm also pretty okay with tagging 1.0 since it's been used for a couple years without complaints.

**Pull request merged**